EasyPost.Tests/EndShipperTest.cs

```diff
@@ -0,0 +1,93 @@
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using EasyPost.Beta;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace EasyPost.Tests
+{
+    [TestClass]
+    public class EndShipperTest
+    {
+        private TestUtils.VCR _vcr;
+
+        // This (and all Beta features) cannot use VCR because of a known conflict between using a VCR/custom HttpClient and re-constructing RestSharp.
+
+        [TestInitialize]
+        public void Initialize()
+        {
+            _vcr = new TestUtils.VCR("end_shipper", TestUtils.ApiKey.Production);
+        }
+
+        [TestMethod]
+        public async Task TestAll()
+        {
+            // _vcr.SetUpTest("all");
```

Comment:
These should be uncommented since the whole test itself is skipped. In my mind, the warning/notice above about the tests in this file should be attached to each test in this file so people know why it's marked ignore. If the reason for ignoring the test appears directly inline with the ignore command, it'll help assist the reader in immediately knowing why (in case they missed the heading at the top of the file).

Similarly, I thought this only broke for 2/4 tests. If so, we should not ignore the two tests that actually worked with VCR here.

Reply:
The version we worked on yesterday actually ended up breaking VCR for most tests. It has since been rewritten to not do that, but as a side-effect, ALL beta tests can no longer use VCR.

Reply:
Ah, too bad 👍 

The thoughts about comments still stands